src/connection.h

```diff
@@ -142,22 +120,20 @@ class Connection : public std::enable_shared_from_this<Connection>
 
 		std::recursive_mutex m_connectionLock;
 
+		std::queue<OutputMessage_ptr> messageQueue;
```

Comment:
I believe that a deque has slightly better performance (didn't test that though) than a list and causes less memory overhead.